consensus-types/blocks/kzg.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	bodyLength    = 13 // The number of elements in the BeaconBlockBody Container for Electra
+	bodyLength    = 12 // The number of elements in the BeaconBlockBody Container for Electra
 	logBodyLength = 4  // The log 2 of bodyLength
 	kzgPosition   = 11 // The index of the KZG commitment list in the Body
 	kzgRootIndex  = 54 // The Merkle index of the KZG commitment list's root in the Body's Merkle tree
```

Comment:
Those values must be adjusted as a field of beacon block body is deleted.

kzgPosition = 10
kzgRootIndex = 52

This test will be passed if then
```
=== RUN   TestInclusionProofs
    assertions.go:144: deneb_test.go:14 Unexpected error: invalid KZG commitment inclusion proof
```